pkgs/development/python-modules/pyfdt/default.nix

```diff
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+}:
+
+buildPythonPackage rec {
+  pname = "pyfdt";
+  version = "0.3";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "pyfdt";
+    sha256 = "sha256:1w7lp421pssfgv901103521qigwb12i6sk68lqjllfgz0lh1qq31";
+  };
+
```

Comment:
This is missing pythonImportsCheck or running tests

Reply:
I've added pythonImportsCheck. The package comes with a tests/ dir but it doesn't seem to exist during checkPhase -- how would I be able to run the tests?